### PR TITLE
Update vehicle installation requirements for consistency and believability.

### PR DESCRIPTION
**Pull request by Extension-Driver**
#### Summary
Bugfixes "Update some vehicle installation requirements for consistency and believability"

#### Purpose of change

The vehicle installation requirements are all over the place. For example, a wooden frame requires 2 mechanics to remove, where it should use the same skill as used in its crafting - Fabrication. Similarly, a cloth board uses Mechanics for installation - where it should use Tailoring and Fabrication.

This change will make it much easier to create crude early game carts and such, and also decouple them from Mechanics, preventing easy grinding of said skill.

In addition to the changed above, removing wooden parts now requires Prying 1 - same as deconstructing furniture via the Construction menu.

In fact, we already have Electronics as a requirement for installing/removing some parts, such as the Solar Panels. I see no reason why this should not extend to other skills too.

#### Describe the solution

Change some parts to use Fabrication or Tailoring where appropriate.

#### Describe alternatives you've considered

Not doing so, and letting people lament their avatar's lack of woodworking knowledge.

#### Testing

Load in game. Check if it works.

#### Additional context

Let me know if I missed soemthing. This is only the first few, obvious ones.

We should really have a tailoring requirement for vehicle parts, plus a woodworking requirement. The tailoring requirement will call for a wooden needle, thread and rags, whereas woodworking would call for a hammering 2 tool, nails and planks.

### NEW VEHICLE REQUIREMENTS:

- Repair requirement should not be >1 level of install removal except in edge cases like solar panels.
- Anything made of wood uses Fabrication, and anything made of cloth uses Tailoring to repair, insall and remove.
- Removing wooden items now requires a Prying 1 tool to remove nails.
- Metal items require Mechanics to repair, like before.

### CODE REVIEW

**Comment by Maleclypse** [comment]
@mlangsdorf I think this change makes sense but I wanted to run it by you.

**Comment by Broken27** [comment]
> Let me know if I missed soemthing. This is only the first few, obvious ones.

Wooden boat hulls and rafts need mechanics instead of frabrication too

File: data/json/vehicleparts/vehicle_parts.json [hunk 1]
```diff
@@ -143,10 +143,10 @@
     "description": "A metal handle, a creature with hands could use it to push a vehicle.",
     "power": 0,
     "location": "structure",
-    "requirements": {
-      "install": { "skills": [ [ "mechanics", 2 ] ], "time": "100 s", "using": [ [ "welding_standard", 5 ] ] },
-      "removal": { "skills": [ [ "mechanics", 1 ] ], "time": "100 s", "using": [ [ "vehicle_weld_removal", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 1 ] ], "time": "20 s", "using": [ [ "welding_standard", 5 ] ] }
+    "requirements":
```
Comment (anothersimulacrum) [inline data/json/vehicleparts/vehicle_parts.json:146]:
```suggestion
    "requirements": {
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 2]
```diff
@@ -868,9 +868,9 @@
     "item": "w_table",
     "location": "center",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 1 ] ], "time": "15 m", "using": [ [ "vehicle_wrench_2", 1 ] ] },
-      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "15 m", "using": [ [ "vehicle_wrench_2", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 2 ] ], "time": "15 m", "using": [ [ "welding_standard", 5 ] ] }
+      "install": { "skills": [ [ "fabrication", 1 ] ], "time": "15 m", "using": [ [ "vehicle_wrench_2", 1 ] ] },
+      "removal": { "skills": [ [ "fabrication", 2 ] ], "time": "15 m", "using": [ [ "vehicle_wrench_2", 1 ] ] },
+      "repair": { "skills": [ [ "fabrication , 2 ] ], "time": "15 m", "using": [ [ "adhesive", 2 ] ] }
```
Comment (kevingranade) [inline data/json/vehicleparts/vehicle_parts.json:873]:
```suggestion
      "repair": { "skills": [ [ "fabrication" , 2 ] ], "time": "15 m", "using": [ [ "adhesive", 2 ] ] }
```

Comment (kevingranade) [inline data/json/vehicleparts/vehicle_parts.json:873]:
```suggestion
      "repair": { "skills": [ [ "fabrication", 2 ] ], "time": "15 m", "using": [ [ "adhesive", 2 ] ] }
```

**Review by I-am-Erk — COMMENTED** [review]
I think these are generally okay. I'm not sure tailoring is the right skill for repairing a cloth board with glue, fabrication would still be fine. I think mechanics should still play a role in some of these wooden parts, particularly the ones with more moving pieces, like a water wheel. You might want to consider downgrading fabrication in those cases.

File: data/json/vehicleparts/vehicle_parts.json [hunk 3]
```diff
@@ -951,10 +951,10 @@
     "durability": 30,
     "item": "raft_board",
     "location": "under",
-    "requirements": {
-      "install": { "skills": [ [ "mechanics", 1 ] ], "time": "30 m", "using": [ [ "rope_natural_short", 2 ] ] },
-      "removal": { "skills": [ [ "mechanics", 1 ] ], "time": "15 m", "qualities": [ { "id": "CUT", "level": 1 } ] },
-      "repair": { "skills": [ [ "mechanics", 2 ] ], "time": "15 m", "using": [ [ "rope_natural_short", 1 ] ] }
+    "requirements":
```
Comment (kevingranade) [inline data/json/vehicleparts/vehicle_parts.json:954]:
```suggestion
    "requirements": {
```

**Comment by Broken27** [comment]
Hmmm, maybe you should add a survival requirement for rafts? I think that would be appropiate.

Also, increase the fab needed to make wooden hulls, 1 is clearly to low.

Fab 4-5 for wooden hulls and Fab 3 Survival 3 for rafts would be about right for me.

File: data/json/vehicleparts/boards.json [hunk 4]
```diff
@@ -60,9 +60,9 @@
     "folded_volume": "3750 ml",
     "breaks_into": "ig_vp_cloth",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 1 ] ], "time": "30 m", "using": [ [ "vehicle_nail_install", 1 ] ] },
-      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "15 m", "using": [ [ "vehicle_nail_removal", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 2 ] ], "time": "30 m", "using": [ [ "adhesive", 2 ] ] }
+      "install": { "skills": [ [ "fabrication", 1 ] ], "time": "30 m", "using": [ [ "vehicle_nail_install", 1 ] ] },
+      "removal": { "skills": [ [ "fabrication", 2 ] ], "time": "15 m", "using": [ [ "vehicle_nail_removal", 1 ] ] },
+      "repair": { "skills": [ [ "tailor", 2 ] ], "time": "30 m", "using": [ [ "adhesive", 2 ] ] }
```
Comment (kevingranade) [inline data/json/vehicleparts/boards.json:65]:
```suggestion
      "repair": { "skills": [ [ "fabrication", 2 ] ], "time": "30 m", "using": [ [ "adhesive", 2 ] ] }
```

File: data/json/vehicleparts/cargo.json [hunk 5]
```diff
@@ -15,9 +15,9 @@
     "item": "bag_canvas",
     "location": "center",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 1 ] ], "time": "30 m", "using": [ [ "vehicle_nail_install", 1 ] ] },
-      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "15 m", "using": [ [ "vehicle_nail_removal", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 2 ] ], "time": "30 m", "using": [ [ "adhesive", 2 ] ] }
+      "install": { "skills": [ [ "fabrication", 1 ] ], "time": "30 m", "using": [ [ "vehicle_nail_install", 1 ] ] },
+      "removal": { "skills": [ [ "fabrication", 2 ] ], "time": "15 m", "using": [ [ "vehicle_nail_removal", 1 ] ] },
+      "repair": { "skills": [ [ "tailoring", 2 ] ], "time": "30 m", "using": [ [ "adhesive", 2 ] ] }
```
Comment (kevingranade) [inline data/json/vehicleparts/cargo.json:20]:
```suggestion
      "repair": { "skills": [ [ "fabrication", 2 ] ], "time": "30 m", "using": [ [ "adhesive", 2 ] ] }
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 6]
```diff
@@ -268,8 +268,8 @@
     "location": "structure",
     "breaks_into": [ { "item": "splinter", "count": [ 3, 5 ] }, { "item": "string_36", "count": [ 10, 15 ] } ],
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 0 ] ], "time": "10 m", "using": [ [ "rope_natural_short", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 1 ] ], "time": "5 m", "using": [ [ "adhesive", 1 ] ] }
+      "install": { "skills": [ [ "fabrication", 0 ] ], "time": "10 m", "using": [ [ "rope_natural_short", 1 ] ] },
+      "repair": { "skills": [ [ "fabrciation", 1 ] ], "time": "5 m", "using": [ [ "adhesive", 1 ] ] }
```
Comment (kevingranade) [inline data/json/vehicleparts/vehicle_parts.json:272]:
```suggestion
      "repair": { "skills": [ [ "fabrication", 1 ] ], "time": "5 m", "using": [ [ "adhesive", 1 ] ] }
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 7]
```diff
@@ -1137,9 +1137,9 @@
     "location": "engine_block",
     "folded_volume": "500 ml",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 1 ] ], "time": "6 m" },
-      "removal": { "skills": [ [ "mechanics", 1 ] ], "time": "6 m" },
-      "repair": { "skills": [ [ "mechanics", 2 ] ], "time": "60 m" }
+      "install": { "skills": [ [ "fabrication", 1 ] ], "time": "6 m" },
+      "removal": { "skills": [ [ "fabrciation", 2 ] ], "time": "6 m" },
```
Comment (kevingranade) [inline data/json/vehicleparts/vehicle_parts.json:1141]:
```suggestion
      "removal": { "skills": [ [ "fabrication", 2 ] ], "time": "6 m" },
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 8]
```diff
@@ -106,9 +106,9 @@
     "power": 0,
     "location": "structure",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 2 ] ], "time": "100 s", "using": [ [ "vehicle_screw", 1 ] ] },
-      "removal": { "skills": [ [ "mechanics", 1 ] ], "time": "100 s", "using": [ [ "vehicle_screw", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 1 ] ], "time": "20 s", "using": [ [ "adhesive", 1 ] ] }
+      "install": { "skills": [ [ "fabriation", 1 ] ], "time": "100 s", "using": [ [ "vehicle_screw", 1 ] ] },
```
Comment (kevingranade) [inline data/json/vehicleparts/vehicle_parts.json:109]:
```suggestion
      "install": { "skills": [ [ "fabrication", 1 ] ], "time": "100 s", "using": [ [ "vehicle_screw", 1 ] ] },
```

File: data/json/vehicleparts/rams.json [hunk 9]
```diff
@@ -125,9 +125,9 @@
     "durability": 312,
     "breaks_into": "ig_vp_wood_plate",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 2 ] ], "time": "60 m", "using": [ [ "vehicle_nail_install", 1 ] ] },
-      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "30 m", "using": [ [ "vehicle_nail_removal", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 3 ] ], "time": "60 m", "using": [ [ "adhesive", 2 ] ] }
+      "install": { "skills": [ [ "fabrication", 2 ] ], "time": "60 m", "using": [ [ "vehicle_nail_install", 1 ] ] },
+      "removal": { "skills": [ [ "fabrication", 2 ] ], "time": "30 m", "using": [ [ "vehicle_nail_removal", 1 ] ] },
+      "repair": { "skills": [ [ "fabircation", 3 ] ], "time": "60 m", "using": [ [ "adhesive", 2 ] ] }
```
Comment (kevingranade) [inline data/json/vehicleparts/rams.json:130]:
```suggestion
      "repair": { "skills": [ [ "fabrication", 3 ] ], "time": "60 m", "using": [ [ "adhesive", 2 ] ] }
```

**Pull request merged**